ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -15,7 +15,7 @@
       - "{{ zookeeper.port }}:2181"
 
 - name: wait until the Zookeeper in this host is up and running
-  action: shell echo ruok | nc -w 3 -i 1 {{ inventory_hostname }} {{ zookeeper.port }}
+  action: shell (echo ruok; sleep 1) | nc {{ inventory_hostname }} {{ zookeeper.port }}
```

Comment:
There's no issue here in terms of user input that needs to be sanitized, is there? Hence the security concerns don't apply; do you disagree?